package.json

```diff
@@ -85,12 +86,12 @@
     "rimraf": "^2.5.2",
     "sanitize.css": "^3.3.0",
     "shelljs": "^0.6.0",
-    "sinon": "^1.17.3",
+    "sinon": "^2.0.0-pre",
```

Comment:
Nevermind, this doesn't work when using sinon@1.17.3 either, I'd still opt to not use a pre-release version of it. This problem is probably Karma and/or webpack related, I'll dig further!